### PR TITLE
unify bibtex entries

### PR DESCRIPTION
**Pull request by tjhei**
follow-up to #2277

### CODE REVIEW

File: doc/manual/manual.bib [hunk 1]
```diff
@@ -11270,15 +11270,17 @@ @Article{rose_freesurface
...
-  Title                    = {High Accuracy Mantle Convection Simulation through Modern Numerical Methods. {II}: {R}ealistic Models and Problems},
-  Author                   = {Heister, Timo and Dannberg, Juliane and Gassm{\"o}ller, Rene and Bangerth, Wolfgang},
-  Journal                  = {Geophysical Journal International},
-  Year                     = {2017},
-  Number                   = {2},
-  Pages                    = {833-851},
-  Volume                   = {210},
-  Doi                      = {10.1093/gji/ggx195},
-  Url                      = {https://doi.org/10.1093/gji/ggx195}
+  title                    = {High Accuracy Mantle Convection Simulation through Modern Numerical Methods.
```
Comment (gassmoeller) [inline doc/manual/manual.bib]: The article above (rose_freesurface uses the same capitalization. Can you unify that as well?

File: doc/manual/manual.bib [hunk 2]
```diff
@@ -11306,19 +11308,19 @@ @misc{aspect-doi-v2.0.0
...
-  Year          = {2017},
-  Month         = {4},
+@article{aspectmanual,
+  title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}},
+  author        = {W. Bangerth and J. Dannberg and R. Gassm{\"o}ller and T. Heister and others},
+  year          = {2018},
+  month         = {5},
   DOI           = {10.6084/m9.figshare.4865333},
-  Note          = {doi:10.6084/m9.figshare.4865333},
+  note          = {doi:10.6084/m9.figshare.4865333},
```
Comment (gassmoeller) [inline doc/manual/manual.bib]: the article He_2017_DG uses the same old capitalization, care to change that as well?

**Comment by tjhei** [comment]
I have to admit that I don't care about changing the capitalization of all entries in the .bib. @bangerth wanted the ones in cite.html to look nice and this PR just makes these entries identical. So I would prefer to merge as is and (if @bangerth really wants to, open an issue)

**Comment by gassmoeller** [comment]
Ah, then I misunderstood the purpose of this PR. Sure then this is fine.

**Comment by bangerth** [comment]
Bibtex adjusts capitalization anyway, so it really doesn't matter. For journal articles, it makes everything lowercase unless you specifically wrap a letter or word in `{...}`.

**Comment by bangerth** [comment]
Oh, then I misunderstood. But I will politely point out that, again, you say that I did something in #2254 that did not actually happen -- been Trumpin' much lately? ;-)

**Comment by tjhei** [comment]
> But I will politely point out that, again, you say that I did something in #2254 that did not actually happen

I am confused. You did change the capitalization of "note", "title", "DOI" etc. in https://github.com/geodynamics/aspect/pull/2254/files#diff-3bedbebe27b0a2462d0928e8c49af301R96 or not?

**Pull request merged**